lib/prosopite.rb

```diff
@@ -12,14 +12,28 @@ class << self
                 :allow_stack_paths,
                 :ignore_queries,
                 :ignore_pauses,
-                :min_n_queries
+                :min_n_queries,
+                :backtrace_cleaner
 
     def allow_list=(value)
       puts "Prosopite.allow_list= is deprecated. Use Prosopite.allow_stack_paths= instead."
 
       self.allow_stack_paths = value
     end
 
+    def backtrace_cleaner
+      return @backtrace_cleaner if defined?(@backtrace_cleaner)
+
+      if Rails.respond_to?(:backtrace_cleaner)
```

Comment:
We can depend on [railties](https://github.com/rails/rails/blob/23e0345fe900dfd7edd6e8e5a7a6bd54b2a7d2ed/railties/lib/rails.rb#L50-L56) and avoid the availability check + fallback here. That will also fix the failing tests due to Rails's constant absence.

Reply:
I was getting errors locally without this, but may have been having to do with how I ran them instead of the code 🤷🏻‍♂️